GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -455,82 +435,12 @@ private void SetRowCountAndSelectRowsIfReady(int rowCount)
-                    // as completed (meaning the background thread will not be restarted) and still contains work items.
-                    if (!_backgroundQueue.IsCompleted || _backgroundQueue.IsEmpty)
-                    {
-                        return;
-                    }
-                }
-
-                try
-                {
-                    CancellationToken timeoutToken;
-                    Func<CancellationToken, Task> backgroundOperation;
-                    CancellationToken backgroundOperationCancellation;
-                    try
-                    {
-                        using (var timeoutTokenSource = new CancellationTokenSource(TimeSpan.FromMilliseconds(200)))
-                        using (var linkedCancellation = timeoutTokenSource.Token.CombineWith(cancellationToken))
-                        {
-                            timeoutToken = timeoutTokenSource.Token;
-                            (backgroundOperation, backgroundOperationCancellation) = await _backgroundQueue.DequeueAsync(linkedCancellation.Token);
-                        }
-                    }
-                    catch (OperationCanceledException) when (timeoutToken.IsCancellationRequested && !cancellationToken.IsCancellationRequested)
-                    {
-                        // No work was received from the queue within the timeout.
-                        if (RowCount < _revisionGraph.Count)
-                        {
-                            this.InvokeAsync(() => { SetRowCountAndSelectRowsIfReady(_revisionGraph.Count); }).FileAndForget();
-                        }
-
-                        continue;
-                    }
-
-                    if (backgroundOperationCancellation.IsCancellationRequested)
-                    {
-                        continue;
-                    }
-
-                    try
-                    {
-                        await backgroundOperation(backgroundOperationCancellation);
-                    }
-                    catch (OperationCanceledException) when (backgroundOperationCancellation.IsCancellationRequested)
-                    {
-                        // Normal cancellation of background work
-                    }
-                }
-                catch (OperationCanceledException) when (_backgroundQueue.IsCompleted && _backgroundQueue.IsEmpty)
-                {
-                    // Normal completion of background work
-                    return;
-                }
-                catch (OperationCanceledException) when (cancellationToken.IsCancellationRequested)
-                {
-                    // Normal cancellation of background queue during clear
-                    return;
-                }
-            }
+            _backgroundUpdater.ScheduleExcecution();
         }
 
-        private void UpdateVisibleRowRange()
+        private async Task UpdateVisibleRowRangeInternal()
```

Comment:
`async` methods should have `Async` suffix.

Reply:
✔ done